Library/Homebrew/manpages/brew.1.md

```diff
@@ -126,6 +126,25 @@ Note that these flags should only appear after a command.
     type dependencies, pass `--skip-build`. Similarly, pass `--skip-optional`
     to skip `:optional` dependencies.
 
+  * `desc` <formula>:
+    Display <formula>'s name and one-line description.
+
+  * `desc [-s|-n|-d] <pattern>`:
+    Search both name and description (`-s`), just the names (`-n`), or just  the
+    descriptions (`-d`) for `<pattern>`. `<pattern>` is by default interpreted
+    as a literal string; if flanked by slashes, it is instead interpreted as a
+    regular expression. Formula descriptions are cached, and the cache is
+    created on the first search, making that search slower than subsequent
+    ones.
+
+  * `desc --create-cache`:
+
+    Force the [re]construction of the name/description cache.
+
+  * `desc --delete-cache`:
+
+    Delete the name/description cache, if it exists.
```

Comment:
I wanted to make it easy for people to opt out of the description-cacheing mechanism if for some reason it was annoying them, as well as to be able to force the cache's regeneration if for some reason the staleness-detection code missed a corner case.

I’ll readily admit that too much minutiae and detail in the man page can be overwhelming, though. How about I omit the man-page entries, but leave the functionality in place for the time being? (I’ll also clean up the blank lines you pointed out above.)